.github/workflows/upgrade-python-requirements.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         env:
           BRANCH: ${{ inputs.branch }}
           DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
-        run: echo "target_branch=$(if ['$BRANCH' = '']; then echo '$DEFAULT_BRANCH'; else echo '$BRANCH'; fi)" >> $GITHUB_ENV
+        run: echo "target_branch=$(if [ $BRANCH = '' ]; then echo $DEFAULT_BRANCH; else echo $BRANCH; fi)" >> $GITHUB_ENV
```

Comment:
You can't have unquoted variables in `[ ]` evaluation because you run into syntax errors if the variable is ever empty.  This is my take on a more resilient fix:

https://github.com/openedx/.github/pull/35